src/migration_utils.py

```diff
@@ -218,10 +234,35 @@ def build_user_object_with_passwords(extracted_user, hash_params):
             )
         ]
         return user_object
+    
+    # Create temporary password if anonymous user
+    elif (not extracted_user["email"]) and (not extracted_user["phone"]):
+        userPasswordToCreate = UserPassword(
+            cleartext="@nonymousPass835"
```

Comment:
We should automatically generate a password with bcrypt and then set it as the `UserPasswordBcrypt` value in Descope, instead of setting all of the passwords the same.